typewise_alert.py

```diff
@@ -1,46 +1,67 @@
+cooling_types_ranges = {
+    "PASSIVE_COOLING": {'lower_limit': 0, 'upper_limit': 35},
+    "MED_ACTIVE_COOLING": {'lower_limit': 0, 'upper_limit': 40},
+    "HI_ACTIVE_COOLING": {'lower_limit': 0, 'upper_limit': 45},
+}
 
-def infer_breach(value, lowerLimit, upperLimit):
-  if value < lowerLimit:
-    return 'TOO_LOW'
-  if value > upperLimit:
-    return 'TOO_HIGH'
-  return 'NORMAL'
-
-
-def classify_temperature_breach(coolingType, temperatureInC):
-  lowerLimit = 0
-  upperLimit = 0
-  if coolingType == 'PASSIVE_COOLING':
-    lowerLimit = 0
-    upperLimit = 35
-  elif coolingType == 'HI_ACTIVE_COOLING':
-    lowerLimit = 0
-    upperLimit = 45
-  elif coolingType == 'MED_ACTIVE_COOLING':
-    lowerLimit = 0
-    upperLimit = 40
-  return infer_breach(temperatureInC, lowerLimit, upperLimit)
-
-
-def check_and_alert(alertTarget, batteryChar, temperatureInC):
-  breachType =\
-    classify_temperature_breach(batteryChar['coolingType'], temperatureInC)
-  if alertTarget == 'TO_CONTROLLER':
-    send_to_controller(breachType)
-  elif alertTarget == 'TO_EMAIL':
-    send_to_email(breachType)
-
-
-def send_to_controller(breachType):
-  header = 0xfeed
-  print(f'{header}, {breachType}')
-
-
-def send_to_email(breachType):
-  recepient = "a.b@c.com"
-  if breachType == 'TOO_LOW':
-    print(f'To: {recepient}')
-    print('Hi, the temperature is too low')
-  elif breachType == 'TOO_HIGH':
-    print(f'To: {recepient}')
-    print('Hi, the temperature is too high')
+email_alert_message = {"TOO_LOW": {'recipient': 'bmslowbreachalert@bosch.com',
```

Comment:
Possibility to give different email recepients for temperature going low or high - is that intentional?
Different persons in charge of heater and cooler :)

Reply:
Hi Sudeep Sir,

Yes, it was intentional as in real use cases there might be two teams taking care of heater and cooler.